fe2-android/app/build.gradle

```diff
@@ -16,179 +16,147 @@ android {
 
         testInstrumentationRunner "com.github.dedis.popstellar.HiltTestRunner"
 
+        // The following feature is not working properly with the coverage on the current build version
+        // As it's purpose is to clear physical data between each tests, its is not used yet.
```

Comment:
`its is`?

Reply:
the 'clearPackage: true'

Reply:
I don't really see how to improve the comments, but I want a comment to explain its purpose and another to explain why it is disabled

Reply:
I should have been clearer. `its is` looks like a typo 😄 
```suggestion
        // Its purpose is to clear physical data between each tests, but it is not used yet.
```

Reply:
And indeed the comment is clear.